src/main/java/com/ericsson/ei/subscriptionhandler/InformSubscription.java

```diff
@@ -102,25 +102,8 @@ public void informSubscriber(String aggregatedObject, JsonNode subscriptionJson)
         String subscriptionName = getSubscriptionField("subscriptionName", subscriptionJson);
         String notificationType = getSubscriptionField("notificationType", subscriptionJson);
         String notificationMeta = getSubscriptionField("notificationMeta", subscriptionJson);
-        String key = "";
-        String val = "";
-        MultiValueMap<String, String> mapNotificationMessage = new LinkedMultiValueMap<>();
-        ArrayNode arrNode = (ArrayNode) subscriptionJson.get("notificationMessageKeyValues");
 
-        if (arrNode.isArray()) {
-            for (final JsonNode objNode : arrNode) {
-                if (objNode.get("formkey").toString().replaceAll(REGEX, "").equals("Authorization")) {
-                    key = "Authorization";
-                    val = objNode.get("formvalue").toString().replaceAll(REGEX, "");
-
-                } else {
-
-                    mapNotificationMessage.add(objNode.get("formkey").toString().replaceAll(REGEX, ""), jmespath
-                            .runRuleOnEvent(objNode.get("formvalue").toString().replaceAll(REGEX, ""), aggregatedObject)
-                            .toString().replaceAll(REGEX, ""));
-                }
-            }
-        }
+        MultiValueMap<String, String> mapNotificationMessage = mapNotificationMessage(aggregatedObject, subscriptionJson);
```

Comment:
reset key and value fields to empty fields at the end of this method and add a comment that we add this because we do not want to leak authentication details.